backend/src/main/java/ca/bc/gov/hlth/mohums/webclient/WebClientService.java

```diff
@@ -26,6 +28,26 @@ public WebClientService(WebClient kcAuthorizedWebClient) {
                 .flatMapMany(r -> r.bodyToFlux(Object.class));
     }
 
+    public Mono<Object> getClient(String clientId) {
+        String path = clientsPath + "/" + clientId;
+        return get(path, null);
+    }
+
+    public Mono<Object> getAssignedUserClientRoleMappings(String userId, String clientId) {
```

Comment:
Took a new approach to writing these webservice methods, with this approach (after a refactor) we'd make the actual get(path, queryParam) method into a private method. 
If we like this approach I'll start a refactor branch for groups and clients to use it as well.